test/NuGet.Core.FuncTests/Dotnet.Integration.Test/Dotnet.Integration.Test.csproj

```diff
@@ -1,13 +1,17 @@
 <Project>
+  <PropertyGroup>
+    <RequiresSigningXplatAPIs>true</RequiresSigningXplatAPIs>
+  </PropertyGroup>
+
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.test.props" />
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
   
   <PropertyGroup>
-    <TargetFrameworks>$(NETCoreTargetFramework)</TargetFrameworks>
+    <TargetFrameworks>$(NETCoreTestTargetFrameworks)</TargetFrameworks>
     <TestProject>true</TestProject>
     <Description>Integration tests for NuGet-powered dotnet CLI commands such as pack/restore/list package and dotnet nuget.</Description>
   </PropertyGroup>
-
+  
```

Comment:
nitpick: trailing whitespace goes against coding guideline 23, and our `.editorconfig` file also instructs compatible editors to remove trailing whitespace, meaning you're using an editor that does not respect `.editorconfig`.

Reply:
Thanks. Will fix in https://github.com/NuGet/NuGet.Client/pull/3571